### ISSUE TITLE
BUG - sev-2 - Probably all? Definitely iOS - Some links not announcing as links to screenreader

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9800]
## What happened?

On at least one link in the mobile app ("opt in to the new decision review process" for an active appeal with a type of `pending_form9`), but I'm willing to bet a few more beyond that, we are not announcing the link as a link to screen reader users. [Here's a video of it in iOS](https://drive.google.com/file/d/1WWDoeX5HEhzgrzuyv68G9Q0DeAdu47yp/view?usp=drive_link) (the only reason I double-tapped on the link was to show that it IS in fact a link, even though it's not announced as one, not because we could reasonably expect screen reader users to do so)

We read the content of the link, and that's it. So screen reader users won't/can't know that the link is a link, and therefore actionable, and do whatever the link is about (in this case, it's one of the MAIN actions they could do for an appeal in this status)

I don't 100% know what's causing this in code, but there are several MobileBodyLink instances in the AppealCurrentStatus file that are at least suspicious-looking, to me, so I'd be willing to bet they are also impacted.

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

* Will need to have an active appeal with a type of pending_form9, or use Charles Proxy to mock one
* Log in as a user with screen reader on, and navigate to ^ appeal
* Swipe through the whole screen, notice the "opt in to the new decision review process" bullet point link is not announced as a link

## Desired behavior

All links should be announced with a role of 'link'

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High 
- Frequency: Low 

- 2 - High

## Linked to Story

Found while testing, but not caused by, #5779

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by wavelaurenrussell** [comment]
@Sparowhawk answering from refinement: yes I would make that the link component so that SR can interact with it and confidently make a choice. If we do that, then we don't need this link within a bulleted list.

**Comment by Sparowhawk** [comment]
For QA the ones that were updated were

statutory_opt_in
pending_form9
merged

those were the only 3 that had links

**Comment by DJUltraTom** [comment]
@TKDickson @Sparowhawk do we have an account with "pending_form9" or a json for the updated links to validate against?

**Comment by TKDickson** [comment]
@DJUltraTom 
afaik we don't have an account - I found the issue by taking an account with appeals (probably Arfan) and modifying the type of an existing appeal to pending_form9

**Comment by DJUltraTom** [comment]
using the on demand build for 9800 the link is still not announcing properly
[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNXpKQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1dd14a1b16d4ab0e2e4b32ddb1ba71a9c76182e7/RPReplay_Final1731958210.MOV)

**Comment by TKDickson** [comment]
Added sample charles response to ticket

**Comment by wavelaurenrussell** [comment]
Yeah, we scoped the Claims work only to Claims. So if you didn't complete any work within Appeals, then that's probably why. Due to the limited scope in Claims.

**Comment by digitalken** [comment]
Checking the following in the test build:

statutory_opt_in
pending_form9
merged

These all announce append "link" to the respective sentences/paragraphs with links in them on iOS and Android.
